### PR TITLE
Ampgw 879

### PR DESCRIPTION
**Pull request by wkolsi**
We have added this flag which is true by default, so that we can on the GovernanceAgent side manage the status, but we keeping the periodic status update.

### CODE REVIEW

**Review by jcollins-axway — DISMISSED** [review]

File: pkg/config/agentfeaturesconfig.go [hunk 1]
```diff
@@ -24,6 +25,7 @@ type AgentFeaturesConfiguration struct {
 	PersistCache            bool              `config:"persistCache"`
 	MarketplaceProvisioning bool              `config:"marketplaceProvisioning"`
 	ExternalIDPConfig       ExternalIDPConfig `config:"idp"`
+	SdkAgentStatusUpdates   bool              `config:"sdkAgentStatusUpdates"`
```
Comment (vivekschauhan) [inline pkg/config/agentfeaturesconfig.go:28]: Can this be changed to enableAgentStatusUpdates?

**Review by jcollins-axway — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

File: pkg/config/agentfeaturesconfig.go (hunk 1 above)
Comment (wkolsi) [reply pkg/config/agentfeaturesconfig.go]: Can we rename it as agentStatusUpdates  without the "enable" to be consistent with all the other flags like (versionChecker, marketplaceProvisioning, connectToCentral , ...)?

File: pkg/config/agentfeaturesconfig.go [hunk 2]
```diff
@@ -90,17 +99,19 @@ func AddAgentFeaturesConfigProperties(props properties.Properties) {
 	props.AddBoolProperty(pathVersionChecker, true, "Controls whether the agent SDK version checker will be enabled or not")
 	props.AddBoolProperty(pathPersistCache, false, "Controls whether the agent SDK will persist agent cache or not")
 	props.AddBoolProperty(pathMarketplaceProvisioning, false, "Controls whether the agent should handle Marketplace Subscriptions or not")
+	props.AddBoolProperty(pathEnableAgentStatusUpdates, false, "Controls whether the agent should manage the status update or not")
```
Comment (vivekschauhan) [inline pkg/config/agentfeaturesconfig.go:102]: The default here should be true

**Review by vivekschauhan — APPROVED** [review]

File: pkg/config/agentfeaturesconfig.go (hunk 1 above)
Comment (jcollins-axway) [reply pkg/config/agentfeaturesconfig.go]: I agree that we should drop enable from the name.

**Review by vivekschauhan — APPROVED** [review]

**Pull request merged**